packages/core/src/Stores/ui-store.js

```diff
@@ -876,6 +879,10 @@ export default class UIStore extends BaseStore {
         this.is_wallet_modal_visible = value;
     }
 
+    toggleIsWalletCreationSuccessModalOpen() {
+        this.is_wallet_creation_success_modal_open = !this.is_wallet_creation_success_modal_open;
+    }
+
```

Comment:
```suggestion
    toggleIsWalletCreationSuccessModalOpen(value) {
        this.is_wallet_creation_success_modal_open = value;
    }
```
Would prefere something like this to be honest. Later in the future it is easier for us to debug.